extension/js/agent/actions/analytics.js

```diff
@@ -0,0 +1,32 @@
+;(function(Agent) {
+
+  var Parse = Agent.Parse;
+  var session_id = Agent.randomString(10);
+
+
+  var saveRecord = function(object, data) {
+    object.save(data, {
+      success: function(object) {
+      },
+      error: function(model, error) {
+      }
+    });
+  }
+
+  Agent.markEvent = function(eventName, data) {
+    var Event = Parse.Object.extend("Event");
+    var event = new Event();
+
+    var eventData = _.extend({
+      event_name: eventName,
+      session_id: session_id,
```

Comment:
not sure, probably usage statistics. i'd like a unique id for the user/browser to say how many ppl use the app